float32 x_center
float32 y_center
float32 cluster_width
float32 cluster_length
bool brake_status
